pkg/R/sparkR.R

```diff
@@ -110,9 +110,19 @@ sparkR.init <- function(
                 mainClass = "edu.berkeley.cs.amplab.sparkr.SparkRBackend",
                 args = as.character(sparkRBackendPort),
                 javaOpts = paste("-Xmx", sparkMem, sep = ""))
-  Sys.sleep(2) # Wait for backend to come up
+
+  cat("Waiting JVM bring up ...\n")
+  while(TRUE) {
+    if(!connExists(.sparkREnv)) {
```

Comment:
Its good to keep it here, but I was just suggesting putting it above this while loop so that, lets say if connectBackend fails and we don't finish the init process. Now if someone calls `sparkR.stop` its good to have the port around to try and stop the backend (if possible).